tasks/interview-corejs.md

```diff
@@ -65,29 +65,29 @@
 
 - #### Functional Patterns
 
-  - Immediately invoked functional expression `(IIFE)`
-  - Know IIFE pattern
+  - Immediately invoked functional expression `(IIFE)` `(optional)`
+  - Know IIFE pattern `(optional)`
   - Callback (Function as argument)
   - Know callback pattern
-  - Understand callback limitations (callback hell)
+  - Understand callback limitations (callback hell) `(optional)`
   - Binding, binding one function twice
```

Comment:
Maybe it should be divided into 2 questions: 1)binding 2) binding one function twice and make second question as optional